h/cli/commands/user.py

```diff
@@ -14,18 +14,24 @@ def user():
 @user.command()
 @click.option('--username', prompt=True)
 @click.option('--email', prompt=True)
+@click.option('--authority')
 @click.password_option()
 @click.pass_context
-def add(ctx, username, email, password):
+def add(ctx, username, email, password, authority):
     """Create a new user."""
     request = ctx.obj['bootstrap']()
 
     signup_service = request.find_service(name='user_signup')
 
-    user = signup_service.signup(username=username,
-                                 email=email,
-                                 password=password)
-    user.activate()
+    signup_args = {}
```

Comment:
It took me a while to figure out why you did it like this (because you don't want to set the authority when it's `None`). Maybe it's easier to understand if you change this to:

```
signup_args = {
    'username': username,
    'email': email,
    'password': password,
    'require_activation': False
}
if authority:
    signup_args['authority'] = authority

signup_service.signup(**signup_args)
```